addon/components/polaris-page/header.js

```diff
@@ -104,4 +115,11 @@ export default Component.extend({
   hasActions: or('primaryAction', 'secondaryActions').readOnly(),
   hasSecondaryActions: gt('secondaryActions.length', 0).readOnly(),
   hasRollup: gt('secondaryActions.length', 1).readOnly(),
+
+  shouldRenderPrimaryActionAsPrimary: computed('primaryAction.primary', function() {
```

Comment:
Very happy to rename this if anyone can think of anything better :wink:

Reply:
Yeah, it's weird that this is a thing now but 🤷‍♂️ I'm good with it